examples/hello_server.cc

```diff
@@ -15,19 +15,24 @@ class HelloHandler : public Http::Handler {
     HTTP_PROTOTYPE(HelloHandler)
 
     void onRequest(const Http::Request& request, Http::ResponseWriter response) {
-        response.send(Http::Code::Ok, "Hello World");
+        (void) request;
+        response.send(Pistache::Http::Code::Ok, "Hello World\n");
     }
 };
 
 int main() {
     Pistache::Address addr(Pistache::Ipv4::any(), Pistache::Port(9080));
-    auto opts = Pistache::Http::Endpoint::options()
-        .threads(1);
+    auto opts = Pistache::Http::Endpoint::options().threads(1);
 
-    Http::Endpoint server(addr);
+    Pistache::Http::Endpoint server(addr);
     server.init(opts);
-    server.setHandler(Http::make_handler<HelloHandler>());
-    server.serve();
+    server.setHandler(Pistache::Http::make_handler<HelloHandler>());
+
+    std::cout << "Starting server. Test with the following command: "
+              << "\"curl http://127.0.0.1:" << addr.port() << "\"\n";
 
+    std::cout << "Press Enter to Exit" << '\n';
```

Comment:
Why did you change the runtime behavior of this test program?

Reply:
This change has nothing to do with the purpose of this PR. I'll revert it if you like.

I changed the it so that it's possible to stop the execution, this makes it possible to use tool like valgrind to check for memory leaks.

Reply:
Please revert this change.